core/services/periodicbackup/backup.go

```diff
@@ -71,13 +72,13 @@ func NewDatabaseBackup(config Config, logger logger.Logger) DatabaseBackup {
 	if config.DatabaseBackupDir() != "" {
 		dir, err := filepath.Abs(config.DatabaseBackupDir())
 		if err != nil {
-			logger.Errorf("Invalid path for DATABASE_BACKUP_DIR (%s) - please set it to a valid directory path", config.DatabaseBackupDir())
+			lggr.Errorf("Invalid path for DATABASE_BACKUP_DIR (%s) - please set it to a valid directory path", config.DatabaseBackupDir())
```

Comment:
This isn't a failure condition?

Reply:
Yes, this turned out to actually be more in depth than I thought. Working on a proper fix.

Reply:
Still no explicit return/error here? Will it error out on the empty `outputParentDir` later or something?

Reply:
Good catch, this should only fail if `os.Getwd()` fails which imo warrants a `Fatal`